packages/xrpl/test/testUtils.ts

```diff
@@ -52,6 +57,22 @@ export function assertResultMatch(
   )
 }
 
+/**
+ * Check that a transaction error validation fails properly.
+ *
+ * @param tx The transaction that should fail validation.
+ * @param validateTx The transaction-specific validation function (e.g. `validatePayment`).
+ * @param errorMessage The error message that should be included in the error.
+ */
+export function assertTxValidationError(
+  tx: any,
```

Comment:
Can you nail down a type for this parameter? I think `Transaction` (or) `SubmitableTransaction` are suitable candidates.

Reply:
No, because this is purposefully checking error cases - so they're often not valid `Transaction`s.